operator/pkg/central/reconciler/reconciler.go

```diff
@@ -42,9 +44,28 @@ func RegisterNewReconciler(mgr ctrl.Manager) error {
 		pkgReconciler.WithReconcilePeriod(extensions.InitBundleReconcilePeriod),
 		pkgReconciler.WithPauseReconcileAnnotation(pauseReconcileAnnotation),
 	}
+
+	opts, err := addSelectorOptionIfNeeded(selector, opts)
+	if err != nil {
+		return err
```

Comment:
I'm afraid just passing `err` all the way without adding any context can result in hard to track errors such as:
`unable to set up Central reconciler: name too long` or similar.

Reply:
Just added a more descriptive error within the `addSelectorOptionIfNeeded` method